build/AzurePipelinesTemplates/MUX-BuildProject-Steps.yml

```diff
@@ -0,0 +1,55 @@
+parameters:
+  solutionPath: ''
+  nugetConfigPath: ''
+  appxPackageDir: '$(build.artifactStagingDirectory)\$(buildConfiguration)\$(buildPlatform)\AppxPackages'
+  buildOutputDir: '$(Build.SourcesDirectory)\BuildOutput'
+  copyFilesToStaging: false
+  publishPath: '$env:Build_ArtifactStagingDirectory'
+
+steps:
+  - task: CmdLine@1
+    displayName: 'Display build machine environment variables'
+    inputs:
+      filename: 'set'
+
+  - task: powershell@2
+    inputs:
+      targetType: filePath
+      filePath: build\Install-WindowsSdkISO.ps1
+      arguments: 17763
+    displayName: 'Install RS5 SDK (17763)'
+
+  - task: NuGetToolInstaller@0
+    displayName: 'Use NuGet 4.9.1'
+    inputs:
+      versionSpec: 4.9.1
+        
+  - task: 333b11bd-d341-40d9-afcf-b32d5ce6f23b@2
+    displayName: 'NuGet restore ${{ parameters.solutionPath }}'
+    inputs:
+      restoreSolution: ${{ parameters.solutionPath }}
+      feedsToUse: config
+      nugetConfigPath: ${{ parameters.nugetConfigPath }}
+
+  - task: VSBuild@1
+    displayName: 'Build solution ${{ parameters.solutionPath }}'
```

Comment:
I don't think it makes sense to parameterize the solution path/ nuget config path etc.
This should be the set of steps specifically for building MUXControls.sln.

For example, the CopyFilesToStagingDir.ps1 script only really makes sense in the context of MUXControls.sln. I would not try to make this overly general.

Reply:
Actually, I see that you are using it for the ReleaseTest sln also.

I think the workflow of:
* Install SDK
* NuGet Restore
* Build sln
Makes sense as an encapsulated unit.
I would remove the CopyFilesToStagingDir.ps1 and PublishBuildArtifacts tasks since they are very context sensitive.

Reply:
Fair enough, I can remove those.